app/models/topic_converter.rb

```diff
@@ -60,31 +60,30 @@ def convert_to_private_message
 
   private
 
+  def posters
+    @posters ||= @topic.posts.where(deleted_at: nil).pluck(:user_id).to_a
```

Comment:
`pluck` gives you an array and you can remove the duplicates by adding a `.distinct`.

Reply:
Nice suggestion, thanks!

Reply:
Isn't `deleted_at: nil` the default? Unless you explicitly say `with_deleted`?